internal/controller/deployment_to_pdb_controller.go

```diff
@@ -65,6 +68,36 @@ func (r *DeploymentToPDBReconciler) handleDeploymentReconcile(ctx context.Contex
 
 			// PDB already exists, nothing to do
 			log.Info("PodDisruptionBudget already exists", "namespace", pdb.Namespace, "name", pdb.Name)
+			pdbWatcher := &myappsv1.PDBWatcher{}
+			e := r.Get(ctx, types.NamespacedName{Name: pdb.Name, Namespace: pdb.Namespace}, pdbWatcher)
+			if e == nil {
```

Comment:
we're alrady pretty nester here time for a funciton UpdateMinAvailableAsNecessary.